### PR TITLE
Set up minimal maven build. Fixed licenses

### PR DESCRIPTION
**Pull request by skabashnyuk**
Set up minimal maven build. Fixed licenses
Work is not finished.
Will be completed in different PR. this PR need to share basic build

### CODE REVIEW

**Comment by JamesDrummond** [comment]
@skabashnyuk @vkuznyetsov  The license information you added on top is causing problems with the "front matter" with "---" on top. I put in ignores for folders with files that had front matter. We could add licence information but it has to be after front matter(second "---").

File: src/main/assets/imgs/logo-eclipseche.svg [hunk 1]
```diff
@@ -1 +1,13 @@
...
...21-2.1,4.43,4.43,0,0,0-.72-1.69,3.48,3.48,0,0,0-1.38-1.13,5.08,5.08,0,0,0-2.17-.41,5.48,5.48,0,0,0-2.31.46,5,5,0,0,0-1.71,1.26,5.85,5.85,0,0,0-1.09,1.91,7.78,7.78,0,0,0-.43,2.41v10.28h-2.15V388.12h2.15Z" transform="translate(-528.72 -372.69)"/><path class="a" d="M740.48,406.68a6.62,6.62,0,0,0,1,2.24,6,6,0,0,0,1.81,1.64,5.06,5.06,0,0,0,2.63.65,5.23,5.23,0,0,0,3.69-1.23,6.47,6.47,0,0,0,1.84-3.28h2.15a8.6,8.6,0,0,1-2.51,4.64A7.49,7.49,0,0,1,746,413a8.2,8.2,0,0,1-3.6-.73,6.62,6.62,0,0,1-2.48-2,8.75,8.75,0,0,1-1.42-3,13.48,13.48,0,0,1-.46-3.57,12.78,12.78,0,0,1,.46-3.42,9.17,9.17,0,0,1,1.42-3,7.11,7.11,0,0,1,2.48-2.12,7.62,7.62,0,0,1,3.6-.8,7,7,0,0,1,6.06,3.09,9.3,9.3,0,0,1,1.35,3.18,13.94,13.94,0,0,1,.34,3.64H740.15A9.53,9.53,0,0,0,740.48,406.68Zm10.66-6.57a6.45,6.45,0,0,0-1.11-2,5.49,5.49,0,0,0-1.74-1.4,5,5,0,0,0-2.32-.53,5.06,5.06,0,0,0-2.36.53,5.5,5.5,0,0,0-1.74,1.4,7,7,0,0,0-1.14,2,9.82,9.82,0,0,0-.56,2.34h11.41A7.8,7.8,0,0,0,751.13,400.11Z" transform="translate(-528.72 -372.69)"/><polygon class="b" points="0 24.29 0 41.53 23.98 55.37 47.95 41.53 47.95 24.29 23.98 38.13 0 24.29"/><polygon class="c" points="23.98 0 0 13.84 0 31.08 23.98 17.24 33.02 22.46 47.95 13.84 23.98 0"/></svg>
```
Comment (mkuznyetsov) [inline src/main/assets/imgs/logo-eclipseche.svg]: new line

**Comment by skabashnyuk** [comment]
license removed from html files

**Comment by JamesDrummond** [comment]
Related to https://github.com/eclipse/che/issues/3510

**Pull request merged**